webauthn/authentication/verify_authentication_response.py

```diff
@@ -69,7 +69,7 @@ def verify_authentication_response(
     """
 
     # FIDO-specific check
-    if bytes_to_base64url(credential.raw_id) != credential.id:
+    if credential.raw_id != base64url_to_bytes(credential.id):
```

Comment:
I flipped this around so any trailing `=` would affect the check, I assume that's a safer comparison anyway?

Reply:
Can you please revert the this change, and the similar one when verifying registration responses? It'll remove the two files completely from this diff, and the rationale for the change got fixed so I don't quite see the need for this to get refactored anymore.

Reply:
Done ✅ What I changed it to seems like a safer choice regardless, but that can be taken in a different PR. If trailing `=` is optional - I'm not familiar with the webauthn spec - the client might send `credential.id` with trailing `=` causing a failed check here. The way I rewrite it trailing `=` or not would not make a difference.

Do you also want me to revert the changes in `webauthn/helpers/bytes_to_base64url.py`? I rewrote the stripping from memory, but not the same way it turns out.

Reply:
I'm fine with the use of `.rstrip("=")` in that method so I'm fine with it getting updated in this PR.